docs/tutorials/learn.md

```diff
@@ -27,10 +27,10 @@ juvix repl
 
 The response should be similar to:
 
-```juvix
+```jrepl
 Juvix REPL version 0.3: https://juvix.org. Run :help for help
 OK loaded: ./.juvix-build/stdlib/Stdlib/Prelude.juvix
-Stdlib.Prelude>
+Stdlib.Prelude> 1 + 3
```

Comment:
Why add `1 + 3` here? This is not a response that one typically sees after just launching the REPL.

Reply:
Thanks, I changed it during the testing phase, same for the feedback back.